draft-ietf-jsonpath-base.md

```diff
@@ -201,6 +215,12 @@ Query:
 Argument:
 : Short name for the JSON data item a JSONPath expression is applied to.
 
+Nodelist:
+: Output of applying a query to an argument: a list of nodes within
```

Comment:
There is a definition for "node" up in line 183.

I'm not entirely sure we need both "item" and "node", but I would ask for your patience to leave this for now until we figure it out.

Reply:
I think "node" warrants its own entry if we're to define "nodelist."  What is a "nodelist" a list of?

Reply:
OK, I can make line 183 into its own entry.

Reply:
Do we also need ...

```
Value:  A JSON value as defined in [RFC8259].
```

It would make your inline ...

```
(Note that the term JSON
   value also implies that this value complies to the definitions in
   [RFC8259], i.e., is indeed a JSON value.)
```
... obsolete.

Reply:
We already say:    The terminology of [RFC8259] applies.
So we have already imported all that terminology.

On the specific note:
A note is by definition redundant; it just alerts the reader to what is said in the note.  I think it is very much worth to say this again, because it reminds people that we are not taking a position on error handling with respect to broken JSON input -- this is just out of scope for the present specification.